integration-tests/test/shared/utils.ts

```diff
@@ -22,6 +22,7 @@ import { Direction, waitForXDomainTransaction } from './watcher-utils'
 import { OptimismEnv } from './env'
 
 export const GWEI = BigNumber.from(1e9)
+export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000'
```

Comment:
Nit: this is definted in ethers, i believe as `utils.AddressZero`

Reply:
Awesome !